src/config/config.go

```diff
@@ -41,38 +41,61 @@ func LoadConfig(filename string) (*Configuration, error) {
 	return &config, nil
 }
 
+// checks if each value in the config file is valid input
 func ValidateConfiguration(config *Configuration) error {
 
-	//check that the fileInputOption is either live or static
 	//convert all strings to lower case to ignore any capitalizations
 	fileInputL := strings.ToLower(config.FileInputOption)
-	outlierL := strings.ToLower(config.OutlierDetectionAlgorithm)
 
-	//added to check if no window size was put in
-	if config.WindowSize == "" {
-		config.WindowSize = "360"
-		fmt.Println("No window size was passed in. The default window size was set to 360")
+	//check that the fileInputOption is either live or static
+	if fileInputL != "live" && fileInputL != "static" {
+		return errors.New("DataOption in config.json must be either 'live' or 'static'")
 	}
 
+	//checks for valid input corresponding to choice of live vs static data analysis
 	if fileInputL == "live" {
 		//require at least 1 subscription
 		if len(config.Subscriptions) == 0 {
 			return errors.New("choosing live data input stream requires to input at least one subscription")
+			return errors.New("choosing live data input stream requires to input at least one subscription")
 		}
 	} else if fileInputL == "static" {
 		//require valid file path
 		_, err := os.Stat(config.StaticFile)
 		if os.IsNotExist(err) {
-			fmt.Println("Please enter a valid pathway to the static file")
+			return errors.New("Invalid pathway to static file: " + err.Error())
```

Comment:
This does not account for if the error is NOT file does not exist. 

We should still return error even if its not what we are expecting.